### ISSUE TITLE
Web UI

### ISSUE THREAD

**Issue by kathmcp** [phoebe-project/phoebe2-ui#11]
Hi again, I am having alot of problems with the Web UI, in that it appears to be constantly dropping connection to the PHOEBE server. Is there something going on just now, or this a common problem?

**Comment by kecnry** [comment]
I do see a number of errors being raised in the server logs (likely by your systems, but they could also be from other users) that are then failing to send through websockets to the web UI with a flask/socketio error `AttributeError: 'Request' object has no attribute 'namespace'`.  These _might_ be related to your dropped connections, but I cannot reproduce a dropped connection on my end without more details.

Its possible that its either a connection issue or specifically tied to the steps you took (if you can consistently reproduce from a fresh case and let me know step-by-step how to do so, I can try to track it down and fix it).  There's also a chance restarting the server might help if the number of cached systems is causing memory issues, but I see there are a few active connections right now, so I don't want to kill those.  I'll keep an eye out for when there are no active connections and restart the server and maybe that will clear things up on its own.

My other suggestion would be to connect it to a local server (which would help with either connection or memory issues).  I remember you having troubles with this before communicating between windows and WSL, but as long as you launch the browser or UI from WSL as well, I don't see why it wouldn't work.  

Lastly, if you're running into a lot of frustrations with the UI as you're attempting more advanced/expensive tasks in PHOEBE, you may want to consider migrating to using the python interface.   The UI is really designed just to help learn the structure of phoebe, but is not fully-featured and not nearly as well-tested as the python interface, and unfortunately isn't able to get as much development attention.  You can always set up a system in the UI, save the bundle, and then load that in python or a jupyter notebook to continue with your science.

**Comment by kecnry** [comment]
Looks like no one was active for about an hour, so I restarted the server.  Let me know if that seems to have helped with the disconnections or not.

**Comment by kathmcp** [comment]
Thanks for doing that. I have tried it again, starting from scratch and I am now getting the "Name ferr is not defined" again!

I have been using a python interface in parallel with the web version, so I am just going to continue with python now I think (I am now having other problems with that but that's another story!) 

Hopefully I will get there in the end!
